include/storage/io.hpp

```diff
@@ -1,17 +1,13 @@
 #ifndef OSRM_STORAGE_IO_HPP_
 #define OSRM_STORAGE_IO_HPP_
 
-#include "contractor/query_edge.hpp"
-#include "extractor/extractor.hpp"
-#include "extractor/original_edge_data.hpp"
-#include "extractor/query_node.hpp"
 #include "util/exception.hpp"
 #include "util/fingerprint.hpp"
 #include "util/simple_logger.hpp"
-#include "util/static_graph.hpp"
 
 #include <boost/filesystem/fstream.hpp>
 #include <boost/iostreams/seek.hpp>
+#include <iostream>
```

Comment:
includes:
- string  # for std::getline
- istream  # for std::istream
- iterator  # for std::istream_iterator

why iostream?